Files/UserControls/StatusCenter.xaml.cs

```diff
@@ -14,79 +16,100 @@
 
 namespace Files.UserControls
 {
-    public sealed partial class StatusCenter : UserControl
+    public sealed partial class StatusCenter : UserControl, IStatusCenterActions
     {
-        public static ObservableCollection<StatusBanner> StatusBannersSource { get; set; } = new ObservableCollection<StatusBanner>();
+        #region Public Properties
+
+        public ObservableCollection<StatusBanner> StatusBannersSource { get; private set; } = new ObservableCollection<StatusBanner>();
+
+        #endregion
+
+        #region Events
 
         public event EventHandler ProgressBannerPosted;
 
+        #endregion
+
+        #region Constructor
+
         public StatusCenter()
         {
             this.InitializeComponent();
         }
 
-        /// <summary>
-        /// Posts a new banner to the Status Center control for an operation.
-        /// It may be used to return the progress, success, or failure of the respective operation.
-        /// </summary>
-        /// <param name="title">Reserved for success and error banners. Otherwise, pass an empty string for this argument.</param>
-        /// <param name="message"></param>
-        /// <param name="initialProgress"></param>
-        /// <param name="status"></param>
-        /// <param name="operation"></param>
-        /// <returns>A StatusBanner object which may be used to track/update the progress of an operation.</returns>
+        #endregion
+
+        #region IStatusCenterActions
+
         public PostedStatusBanner PostBanner(string title, string message, float initialProgress, ReturnResult status, FileOperationType operation)
         {
             StatusBanner item = new StatusBanner(message, title, initialProgress, status, operation);
             StatusBannersSource.Add(item);
             ProgressBannerPosted?.Invoke(this, EventArgs.Empty);
-            return new PostedStatusBanner(item);
+            return new PostedStatusBanner(item, this);
         }
 
-        /// <summary>
-        /// Posts a new banner with expanded height to the Status Center control. This is typically
-        /// used to represent a failure during a prior operation which must be acted upon.
-        /// </summary>
-        /// <param name="title"></param>
-        /// <param name="message"></param>
-        /// <param name="primaryButtonText"></param>
-        /// <param name="cancelButtonText"></param>
-        /// <param name="primaryAction"></param>
-        /// <returns>A StatusBanner object which may be used to automatically remove the banner from UI.</returns>
```

Comment:
Can you restore the documentation?

Reply:
@yaichenbaum it's been moved to `IStatusCenterActions` interface

Reply:
![image](https://user-images.githubusercontent.com/53011783/110361298-1b58d180-8040-11eb-90bc-720c79958ff1.png)

@yaichenbaum